suzieq/poller/worker/nodes/node.py

```diff
@@ -1340,6 +1347,10 @@ async def init_boot_time(self):
     # pylint: disable=unused-argument
     async def _parse_boottime_hostname(self, output, cb_token) -> None:
         '''Parse the version for uptime and hostname'''
+        if not isinstance(output, list):
```

Comment:
Can you please add a comment where we comment what we are doing here?

Reply:
Added a comment that this is to handle errors.